<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuarantineStyWindow</class>
 <widget class="QMainWindow" name="QuarantineStyWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>576</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Minecraft</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>512</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Minecraft</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quarantine Sty</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_pig_information">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>472</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Minecraft</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="MyPushButton" name="button_dispose">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Minecraft</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dispose all</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_pig">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">MyPushButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
